src/components/signup-card/style.less

```diff
@@ -1,15 +1,17 @@
 @import '../../styles/vars.less';
-@import '../../styles/fonts.css';
+@import '../../styles/fonts.less';
 @import '../../styles/mixins.less';
 
 // Signup Card
 .card-signup{
-  width: @large-card-width;
-  height: @large-card-height;
+  width: 100%;
+  height: 100%;
```

Comment:
We should probably remove some of these variables if we are just going to use percentage units.

Reply:
Yes, we can just let them stay during testing and remove them later